CONTRIBUTORS.md

```diff
@@ -531,7 +533,6 @@ GitHub contributors:
 - Priyank Desai
 - ProBackup-nl
 - Profileo
-- Progi1984
```

Comment:
Was it a duplicate ?

Reply:
Affirmative, my real name is indicated...

Reply:
I thought Franck was your nickname 😮 and Progi1984 your real name !

Reply:
Shut up, it's a secret !